components/frontend/src/source/SourceEntities.css

```diff
@@ -0,0 +1,7 @@
+.ui.sortable.table.entities.stickyHeader > thead {
+    /* Make thead sticky by positioning the th's */
+    position: sticky;
+    /* Leave room for the menu bar, the subject title, and the subject table header row */
+    top: 187px;
```

Comment:
Hmm, there's so many magic numbers in various files :( I guess it can't be helped but I can imagine it could become a dark rabbit hole at some point

Reply:
Yeah, this one should be calculated based on the height of various elements, but I have no idea how to do that in CSS.